test_sets/smoke_tests.txt

```diff
@@ -10,4 +10,10 @@
 ./test_scripts/Smoke/Resumption/ATF_Resumption_big_amount_of_data.lua
 ./test_scripts/Smoke/Resumption/ATF_Resumption_disconnect_more_than_30s_before_SUSPEND.lua
 ./test_scripts/Smoke/Resumption/ATF_Resumption_heartbeat_disconnect.lua
-./test_scripts/Smoke/Resumption/ATF_Resumption_unexpected_disconnect.lua
\ No newline at end of file
+./test_scripts/Smoke/Resumption/ATF_Resumption_unexpected_disconnect.lua
+./test_scripts/Smoke/Registration/ATF_Register_App.lua
+./test_scripts/Smoke/Registration/ATF_Register_5_session.lua
+./test_scripts/Smoke/Registration/ATF_Register_5_connection.lua
+./test_scripts/Smoke/Registration/ATF_Reregister_App.lua
+./test_scripts/Smoke/Registration/ATF_Reregister_App_after_disconnect.lua
+./test_scripts/Smoke/Registration/ATF_Reregister_App_if_two_apps_are_registered.lua
```

Comment:
@LitvinenkoIra, Please add new line after the last script. It is required to run the tests on CI